go.mod

```diff
@@ -106,15 +106,15 @@ require (
 	go.uber.org/atomic v1.10.0 // indirect
 	go.uber.org/multierr v1.11.0 // indirect
 	go.uber.org/zap v1.19.0 // indirect
-	golang.org/x/crypto v0.14.0 // indirect
+	golang.org/x/crypto v0.22.0 // indirect
 	golang.org/x/exp v0.0.0-20220722155223-a9213eeb770e // indirect
 	golang.org/x/mod v0.10.0 // indirect
-	golang.org/x/net v0.17.0 // indirect
+	golang.org/x/net v0.24.0 // indirect
```

Comment:
kube in 1.28 uses /x/net in v0.23.0 - https://github.com/kubernetes/kubernetes/blob/release-1.28/go.mod#L82

why not to match kube ?

Reply:
newer seemed better :shrug:

Reply:
should be backward compatible.